docs/src/forc_client.md

```diff
@@ -2,24 +2,26 @@
 
 Forc plugin for interacting with a Fuel node.
 
+## Initializing the wallet and adding accounts
+
+If you don't have an initialized wallet or any account for your wallet you won't be able to sign transactions.
+
+To initialize a wallet you can use `forc wallet init`. It will ask you to choose a password to encrypt your wallet. After the initialization is done you will have your mnemonic phrase.
+
+After you have an initialized wallet, you can create an account for it by simply running `forc wallet new`. It will ask your password to decrypt the wallet before creating an account.
+
 ## Signing transactions using `forc-wallet` CLI
 
-To submit the transactions created by `forc deploy` or `forc run`, you need to sign them first (unless you are using a client without utxo validation). To sign a transaction you can use `forc-wallet` CLI. This section is going to walk you through the whole signing process.
+To submit the transactions created by `forc deploy` or `forc run`, you need to sign them first (unless you are using a client without UTXO validation). To sign a transaction you can use `forc-wallet` CLI. This section is going to walk you through the whole signing process.
+
+> Disabling the UTXO validation, enables to send invalid inputs to emulate different conditions and by default `fuel-core` runs without UTXO validation. This means while your client is running without UTXO validation, you can run unsigned transactions. If you want to run `fuel-core` with UTXO validation, you can pass `--utxo-validation` to `fuel-core run`. If UTXO valdiation is enabled, unsigned transactions will return an error.
```

Comment:
```suggestion
By default `fuel-core` runs without UTXO validation, which means you can run unsigned transactions. This allows you to send invalid inputs to emulate different conditions.

If you want to run `fuel-core` with UTXO validation, you can pass `--utxo-validation` to `fuel-core run`. If UTXO validation is enabled, unsigned transactions will return an error.
```

I think re-ordering this will help better contextualize the role of `fuel-core` here, if that was your goal